### DISCUSSION TITLE
Phi-3-mini (128k) model

### DISCUSSION THREAD

**Discussion by singhhimanshu97** [community/community#138530] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The model output for a question is vague.
![Screenshot 2024-09-13 093412](https://github.com/user-attachments/assets/c723c8a3-2d8b-4ce3-9c9e-2e46e98c9bc7)

**Comment by matthewisabel** [comment]
Hmm, I've opened an issue here but it certainly looks like an odd output!

Does this happen consistently?

**Reply by singhhimanshu97** [reply]
For some of the prompts yes it happens

**Comment by martincai** [comment]
Hi @singhhimanshu97 
Is this reproducible for you?  Can you please share the full context of the conversation so I can attempt to reproduce it at my end?

**Reply by singhhimanshu97** [reply]
I put a prompt: Are you aware of particle swarm optimization algorithm
The response was yes .......
Next prompt: what is particle swarm optimization.
Response is attached in the image.
